### DISCUSSION TITLE
traffic.tar

### DISCUSSION THREAD

**Discussion by Michkail** [gis-ops/docker-valhalla#158] [General]
after execute thid code:
docker run -dt --name valhalla_gis-ops -p 8002:8002 -v /home/user/custom_files:/custom_files ghcr.io/gis-ops/docker-valhalla/valhalla:latest

i got this:
2024/05/27 02:29:25.323161 [WARN] (stat): /custom_files/traffic.tar No such file or directory
2024/05/27 02:29:25.323175 [WARN] Traffic tile extract could not be loaded

how i got traffic.tar?

**Comment by nilsnolde** [comment]
that's out of scope, you can look at https://github.com/valhalla/valhalla/issues and discussions, lots of info sprinkled around.
